### DISCUSSION TITLE
migration: tools & functions not working from v0.3.17

### DISCUSSION THREAD

**Discussion by tjbck** [open-webui/open-webui#5181] [General]
replace `from utils` to `from open_webui.utils`

**Comment by Goekdeniz-Guelmez** [comment]
Is it possible to downgrade? Because I can't manually set the utils in `/app/backend/data/functions/anthropic.py`, it resets it because of the new image.

**Comment by tjbck** [comment]
@Goekdeniz-Guelmez 0.3.18 should resolve this issue, let me know if the issue persists!

**Comment by Goekdeniz-Guelmez** [comment]
It worked!!! Thanks!
